lib/Pod/To/HTML.pm6

```diff
@@ -11,37 +11,67 @@ class TOC::Calculator {...}
 
 my constant $MAX-COMPILE-TEMPLATE-ATTEMPTS = 5;
 
+role Pod::To::HTML::Renderer {
+    method render(%context) {
+        ...
+    }
+}
+
+class Pod::To::HTML::Mustache does Pod::To::HTML::Renderer {
+    has $.template;
+    has $.main-template-path;
+
+    method render(%context) {
+        # get 'main.mustache' file (and possible its partials) under template path.
+        my ($template-file, %partials) = retrieve-templates($!template, $!main-template-path);
+        my $content = $template-file.IO.slurp;
+
+        # Try to compile a template N times to workaround issues with race-y compilation
```

Comment:
Can't you think of a faster way? This is going to make it way slow.

Reply:
Not really. It was introduced at https://github.com/Raku/Pod-To-HTML/pull/83

The explanation is also given there. In short:

- When the pages are rendered correctly at a first attempt, no slowdown.
- When a bug happens, we re-compile and indeed lose some time, but at least it still works in a concurrent environment and concurrency gives us more speedup that we can lose.

We already had this code, I just moved it.